src/cluster/pager.rs

```diff
@@ -113,7 +113,7 @@ impl<
 
 pub struct QueryPager<'a, Q: ToString, P: 'a> {
     pager: &'a mut P,
-    pager_state: PagerState,
+    pub pager_state: PagerState,
```

Comment:
I want to send clients the bytes. The clients can send the bytes back to retrieve the next page, thats why this needs to be public

Reply:
pub is missing for ExecPager.